test/functional/basic/find-e2e-specs.js

```diff
@@ -121,6 +77,38 @@ describe('XCUITestDriver - find', function () {
     });
   });
 
+  describe('by id', () => {
+    it('should find a single element by id', async () => {
```

Comment:
I just see that having context _this_ available for tests provides better possibilities. So why should we refuse them?